[2018-04-20 09:02:17,863] TRACE [Controller id=1] Checking need to trigger auto leader balancing (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] DEBUG [Controller id=1] Preferred replicas by broker Map(2 -> Map(wing-binlog-event-1 -> Vector(2, 0, 1)), 1 -> Map(wing-binlog-event-0 -> Vector(1, 2, 0)), 0 -> Map(__consumer_offsets-49 -> Vector(0), __consumer_offsets-38 -> Vector(0), __consumer_offsets-16 -> Vector(0), __consumer_offsets-27 -> Vector(0), __consumer_offsets-8 -> Vector(0), __consumer_offsets-19 -> Vector(0), __consumer_offsets-13 -> Vector(0), __consumer_offsets-2 -> Vector(0), __consumer_offsets-46 -> Vector(0), __consumer_offsets-35 -> Vector(0), __consumer_offsets-24 -> Vector(0), __consumer_offsets-5 -> Vector(0), __consumer_offsets-43 -> Vector(0), __consumer_offsets-21 -> Vector(0), __consumer_offsets-32 -> Vector(0), __consumer_offsets-10 -> Vector(0), __consumer_offsets-37 -> Vector(0), __consumer_offsets-48 -> Vector(0), __consumer_offsets-29 -> Vector(0), __consumer_offsets-18 -> Vector(0), __consumer_offsets-40 -> Vector(0), __consumer_offsets-7 -> Vector(0), __consumer_offsets-45 -> Vector(0), __consumer_offsets-23 -> Vector(0), __consumer_offsets-34 -> Vector(0), __consumer_offsets-26 -> Vector(0), __consumer_offsets-4 -> Vector(0), __consumer_offsets-15 -> Vector(0), __consumer_offsets-42 -> Vector(0), __consumer_offsets-31 -> Vector(0), __consumer_offsets-9 -> Vector(0), __consumer_offsets-20 -> Vector(0), __consumer_offsets-12 -> Vector(0), __consumer_offsets-1 -> Vector(0), __consumer_offsets-28 -> Vector(0), __consumer_offsets-17 -> Vector(0), __consumer_offsets-6 -> Vector(0), __consumer_offsets-39 -> Vector(0), __consumer_offsets-44 -> Vector(0), __consumer_offsets-36 -> Vector(0), __consumer_offsets-47 -> Vector(0), __consumer_offsets-3 -> Vector(0), __consumer_offsets-25 -> Vector(0), __consumer_offsets-14 -> Vector(0), __consumer_offsets-30 -> Vector(0), __consumer_offsets-41 -> Vector(0), wing-binlog-event-2 -> Vector(0, 1, 2), __consumer_offsets-11 -> Vector(0), __consumer_offsets-33 -> Vector(0), __consumer_offsets-22 -> Vector(0), __consumer_offsets-0 -> Vector(0))) (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] DEBUG [Controller id=1] Topics not in preferred replica for broker 2 Map() (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] TRACE [Controller id=1] Leader imbalance ratio for broker 2 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] DEBUG [Controller id=1] Topics not in preferred replica for broker 1 Map() (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] TRACE [Controller id=1] Leader imbalance ratio for broker 1 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] DEBUG [Controller id=1] Topics not in preferred replica for broker 0 Map() (kafka.controller.KafkaController)
[2018-04-20 09:02:18,040] TRACE [Controller id=1] Leader imbalance ratio for broker 0 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:55:10,766] DEBUG [Controller id=1] Sending MetadataRequest to Brokers: Vector(0, 1, 2) for TopicPartitions: ArrayBuffer(wing-binlog-event-1) (kafka.controller.KafkaController)
[2018-04-20 09:55:10,828] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:10,828] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:10,828] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:10,954] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:10,958] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:10,958] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,061] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,089] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,067] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,190] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,200] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,201] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,312] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,313] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,313] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,417] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,417] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,417] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,518] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,518] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,525] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,620] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,623] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,629] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,723] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,725] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,733] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,824] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,827] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,835] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,932] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,934] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:11,940] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:12,035] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:12,036] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:12,046] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:271)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:19,378] DEBUG [Controller id=1] Sending MetadataRequest to Brokers: Vector(0, 1, 2) for TopicPartitions: ArrayBuffer(wing-binlog-event-2) (kafka.controller.KafkaController)
[2018-04-20 09:55:19,425] DEBUG [Controller id=1] Sending MetadataRequest to Brokers: Vector(0, 1, 2) for TopicPartitions: ArrayBuffer(wing-binlog-event-0) (kafka.controller.KafkaController)
[2018-04-20 09:55:25,989] DEBUG [Controller id=1] Sending MetadataRequest to Brokers: Vector(0, 1, 2) for TopicPartitions: ArrayBuffer(wing-binlog-event-1) (kafka.controller.KafkaController)
[2018-04-20 09:55:42,148] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:42,148] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:55:42,157] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:12,266] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:12,267] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:12,266] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:42,376] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:42,376] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:56:42,376] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:12,486] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:12,486] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:12,488] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:42,594] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:42,594] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:57:42,594] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:12,704] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:12,704] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:12,704] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:42,815] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:42,816] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:58:42,816] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:12,928] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:12,930] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:12,930] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:23,614] TRACE [Controller id=1] Checking need to trigger auto leader balancing (kafka.controller.KafkaController)
[2018-04-20 09:59:23,619] DEBUG [Controller id=1] Preferred replicas by broker Map(2 -> Map(wing-binlog-event-1 -> Vector(2, 0, 1)), 1 -> Map(wing-binlog-event-0 -> Vector(1, 2, 0)), 0 -> Map(__consumer_offsets-49 -> Vector(0), __consumer_offsets-38 -> Vector(0), __consumer_offsets-16 -> Vector(0), __consumer_offsets-27 -> Vector(0), __consumer_offsets-8 -> Vector(0), __consumer_offsets-19 -> Vector(0), __consumer_offsets-13 -> Vector(0), __consumer_offsets-2 -> Vector(0), __consumer_offsets-46 -> Vector(0), __consumer_offsets-35 -> Vector(0), __consumer_offsets-24 -> Vector(0), __consumer_offsets-5 -> Vector(0), __consumer_offsets-43 -> Vector(0), __consumer_offsets-21 -> Vector(0), __consumer_offsets-32 -> Vector(0), __consumer_offsets-10 -> Vector(0), __consumer_offsets-37 -> Vector(0), __consumer_offsets-48 -> Vector(0), __consumer_offsets-29 -> Vector(0), __consumer_offsets-18 -> Vector(0), __consumer_offsets-40 -> Vector(0), __consumer_offsets-7 -> Vector(0), __consumer_offsets-45 -> Vector(0), __consumer_offsets-23 -> Vector(0), __consumer_offsets-34 -> Vector(0), __consumer_offsets-26 -> Vector(0), __consumer_offsets-4 -> Vector(0), __consumer_offsets-15 -> Vector(0), __consumer_offsets-42 -> Vector(0), __consumer_offsets-31 -> Vector(0), __consumer_offsets-9 -> Vector(0), __consumer_offsets-20 -> Vector(0), __consumer_offsets-12 -> Vector(0), __consumer_offsets-1 -> Vector(0), __consumer_offsets-28 -> Vector(0), __consumer_offsets-17 -> Vector(0), __consumer_offsets-6 -> Vector(0), __consumer_offsets-39 -> Vector(0), __consumer_offsets-44 -> Vector(0), __consumer_offsets-36 -> Vector(0), __consumer_offsets-47 -> Vector(0), __consumer_offsets-3 -> Vector(0), __consumer_offsets-25 -> Vector(0), __consumer_offsets-14 -> Vector(0), __consumer_offsets-30 -> Vector(0), __consumer_offsets-41 -> Vector(0), wing-binlog-event-2 -> Vector(0, 1, 2), __consumer_offsets-11 -> Vector(0), __consumer_offsets-33 -> Vector(0), __consumer_offsets-22 -> Vector(0), __consumer_offsets-0 -> Vector(0))) (kafka.controller.KafkaController)
[2018-04-20 09:59:23,624] DEBUG [Controller id=1] Topics not in preferred replica for broker 2 Map() (kafka.controller.KafkaController)
[2018-04-20 09:59:23,625] TRACE [Controller id=1] Leader imbalance ratio for broker 2 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:59:23,625] DEBUG [Controller id=1] Topics not in preferred replica for broker 1 Map() (kafka.controller.KafkaController)
[2018-04-20 09:59:23,627] TRACE [Controller id=1] Leader imbalance ratio for broker 1 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:59:23,627] DEBUG [Controller id=1] Topics not in preferred replica for broker 0 Map() (kafka.controller.KafkaController)
[2018-04-20 09:59:23,627] TRACE [Controller id=1] Leader imbalance ratio for broker 0 is 0.0 (kafka.controller.KafkaController)
[2018-04-20 09:59:43,035] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9092 (id: 0 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:43,036] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9094 (id: 2 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-20 09:59:43,035] WARN [RequestSendThread controllerId=1] Controller 1's connection to broker 192.168.0.105:9093 (id: 1 rack: null) was unsuccessful (kafka.controller.RequestSendThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:272)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:225)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
